pkg/recommender.py

```diff
@@ -92,6 +92,7 @@ def update(self, *args, **kargs):
                       'url': 'http://hcdm4.cs.virginia.edu:8989'}
```

Comment:
is this just for testing?

Reply:
yes, because people cannot access our server our of UVa wifi.... we should deploy it to the AWS EC2 later

Reply:
got it, thanks!